api/filter/mongo_proxy.proto

```diff
@@ -11,4 +13,10 @@ message MongoProxy {
   // path is specified no access logs will be written. Note that access log is
   // also gated by runtime.
   string access_log = 2;
+
+  // Inject a fixed delay before proxying a Mongo operation. Delays are
+  // applied to the following MongoDB operations: Query, Insert, GetMore,
+  // and KillCursors. Once an active delay is in progress, all incoming
+  // data up until the timer event fires will be a part of the delay.
+  Delay delay = 3;
```

Comment:
This should be `HttpFaultDelay`, or move `Delay` to some neutral shared proto and rename `FaultDelay`.